app/src/main/java/app/revanced/tiktok/settingsmenu/TikTokSettingsFragment.java

```diff
@@ -25,10 +25,10 @@
 import app.revanced.tiktok.settingsmenu.preference.categories.FeedFilterPreferenceCategory;
 import app.revanced.tiktok.settingsmenu.preference.categories.IntegrationsPreferenceCategory;
 import app.revanced.tiktok.settingsmenu.preference.categories.SimSpoofPreferenceCategory;
-import app.revanced.tiktok.utils.ReVancedUtils;
+import app.revanced.tiktok.utils.Utils;
 
 @SuppressWarnings("deprecation")
-public class ReVancedPreferenceFragment extends PreferenceFragment {
+public class TikTokSettingsFragment extends PreferenceFragment {
```

Comment:
Since eventually the integrations will be on a patch and app basis, there is no particular need to prefix the classes according to the app